src/neuroconv/datainterfaces/behavior/video/videodatainterface.py

```diff
@@ -37,12 +43,15 @@ def __init__(self, file_paths: list, verbose: bool = False):  # TODO - debug why
         file_paths : list of FilePathTypes
             Many video storage formats segment a sequence of videos over the course of the experiment.
             Pass the file paths for this videos as a list in sorted, consecutive order.
+        metadata_key_name : str, optional, default: "Videos"
+            The key name to use for the metadata associated with this video interface.
```

Comment:
```suggestion
            The key name to use for the metadata associated with this video interface. This should be located in metadata["Behavior"] ...
```

Can you expand a bit on the docs like that to give more information?

Reply:
ping @luiztauffer